addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -601,6 +601,14 @@
 						<param name="label" value="$LOCALIZE[31123]" />
 						<param name="visible" value="!String.IsEmpty(ListItem.Director)" />
 					</include>
+					<include content="InfoDialogButton">
+						<param name="id" value="999" />
+						<param name="icon" value="DefaultMusicAlbums.png" />
+						<param name="onclick_1" value="Dialog.Close(all,true)" />
+						<param name="onclick_2" value="ActivateWindow(Music,musicdb://albums/?artist=$INFO[ListItem.Artist]&albumartistonly=true&compilation=false)" />
```

Comment:
the `&` character needs to be escaped in xml: `&amp;`

Reply:
Didn't know that, but fixed.  Thank you.